api/resolvers/user.js

```diff
@@ -622,6 +622,19 @@ export default {
   },
 
   Mutation: {
+    disableFreebies: async (parent, args, { me, models }) => {
+      if (!me) {
+        throw new GraphQLError('you must be logged in', { extensions: { code: 'UNAUTHENTICATED' } })
+      }
+
+      // disable freebies if it hasn't been set yet
+      models.user.update({
+        where: { id: me.id, disableFreebies: null },
+        data: { disableFreebies: true }
+      }).catch()
```

Comment:
maybe better to await this since we don't await on the client so the client could print the error?

Reply:
I call this in `enablePayments` though. I thought erroring about a side effect might be weird.

Reply:
Yes but you don't await it there, you only log errors so it wouldn't error on error

Reply:
Good point